src/Query/TimeoutExecutor.php

```diff
@@ -12,8 +12,15 @@ final class TimeoutExecutor implements ExecutorInterface
     private $loop;
     private $timeout;
 
-    public function __construct(ExecutorInterface $executor, $timeout, LoopInterface $loop = null)
+    /**
+     * @param ?LoopInterface $loop
```

Comment:
Should we also add the other types here as suggested in #223?

```suggestion
     * @param ExecutorInterface $executor
     * @param float $timeout
     * @param ?LoopInterface $loop
```